flightgoggles_uav_dynamics:
  init_pose: [0.3, 52.0, 2.5, 0.0, 0.0, -0.707, 0.707]
challenge_name: "AlphaPilot Test 3"
timeout: 300.0
gate_names: ['Gate10', 'Gate21', 'Gate2', 'Gate13', 'Gate9', 'Gate14', 'Gate1', 'Gate22', 'Gate15', 'Gate23', 'Gate6']
gate_width: 0.3
results_location: "results.yaml"
